ksqldb-functional-tests/src/test/resources/historical_plans/average_-_calculate_average_in_select/5.5.0_1579766275923/plan.json

```diff
@@ -7,22 +7,16 @@
       "sourceName" : "TEST",
       "schema" : "`ROWKEY` BIGINT KEY, `ID` BIGINT, `NAME` STRING, `VALUE` BIGINT",
       "keyField" : "ID",
-      "timestampColumn" : null,
       "topicName" : "test_topic",
       "formats" : {
         "keyFormat" : {
-          "format" : "KAFKA",
-          "properties" : { }
+          "format" : "KAFKA"
```

Comment:
This is a single, example, rewritten plan.  I've included this to demonstrate the differences between existing plans and ones that will be written from now on.  All other historical plans have been left untouched, as these represent the plans that will be in the command topics of clusters out in the wild.